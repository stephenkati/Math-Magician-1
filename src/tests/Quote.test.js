import renderer from 'react-test-renderer';
import FetchQuote from '../api/Quote';

describe('test if quote matches snapshot', () => {
  test('test if quote is a match', () => {
    const quoteComponent = renderer
      .create(<FetchQuote />)
      .toJSON();
    expect(quoteComponent).toMatchSnapshot();
  });
});
