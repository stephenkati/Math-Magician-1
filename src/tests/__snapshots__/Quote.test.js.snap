// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if quote matches snapshot test if quote is a match 1`] = `
<div
  className="quote"
/>
`;
